Prox/Prox/Data/PlacesController.swift

```diff
@@ -44,12 +44,12 @@ class PlacesProvider {
         database.getPlace(forKey: key).upon { callback($0.successResult() )}
     }
 
-    func placeWithEvent(forKey key: String, callback: @escaping (Place?) -> ()) {
+    func place(withKey placeKey: String, forEventWithKey eventKey: String, callback: @escaping (Place?) -> ()) {
         let eventProvider = EventsProvider()
         var placeWithEvent: Place?
         var eventForPlace: Event?
         let lock = NSLock()
-        place(forKey: key) { place in
+        place(forKey: placeKey) { place in
```

Comment:
If the place or event cannot be found, we never call the callback.

I think it's unlikely to happen since we wouldn't have received an event with a bad ID or an invalid placeID so it's probably not worth fixing, but perhaps add a comment warning about this possibility.